content/general/concepts/sdlc/sdlc.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'SDLC'
```

Comment:
I wonder if the title should be "Software Development Life Cycle" rather than "SDLC" as someone scanning the general concepts may not be familiar with the acronym.

Reply:
Should we also (with general reorganization efforts) change other acronyms like URN and WSDL into their full form? cc @caupolicandiaz

Reply:
@yangc95, I guess it depends on how its referred to in general discourse. For instance I don't know if it would make sense for XML or WDSL.

Reply:
I think I agree with Steve that we should use the full phrase in this case.